modules/monitoring-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-limits-and-resource-requests-for-monitoring-components_{context}"]
+= Specifying limits and requests for monitoring components 
+
+To configure CPU and memory resources, specify values for resource limits and requests in the appropriate `ConfigMap` object for the namespace in which the monitoring component is located:
+
+* The `cluster-monitoring-config` config map in the `openshift-monitoring` namespace for core platform monitoring.
+* The `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace for components that monitor user-defined projects.
+
+.Prerequisites
+
+* *If you are configuring core platform monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` cluster role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring components that monitor user-defined projects*:
+** You have access to the cluster as a user with the `cluster-admin` cluster role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. To configure core platform monitoring components, edit the `cluster-monitoring-config` config map object in the `openshift-monitoring` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add values to define resource limits and requests for each core platform monitoring component you want to configure.
++
+[WARNING]
```

Comment:
Per the SSG, a warning is appropriate for "Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition. Explain the problem, cause, and offer a solution that works. If available, offer information to avoid the problem in the future or state where to find more information."

If the value set for a limit is not higher than the value set for a request, will that cause system damage, data loss, or a support-related issue? If not, consider changing the warning to an IMPORTANT admonition.

Reply:
Agree. Changing to an Important admonition. Good catch! Thanks.